tools/bonanza/src/Bonanza/Parser/Time.hs

```diff
@@ -88,20 +89,20 @@ commonLogDate = do
   let ld = fromGregorian y m d
       lt = TimeOfDay h m' (fromInteger s)
       tz = hoursToTimeZone z
-   in return $ localTimeToUTC tz (LocalTime ld lt)
+   in pure $ localTimeToUTC tz (LocalTime ld lt)
   where
-    month "Jan" = return 1
-    month "Feb" = return 2
-    month "Mar" = return 3
-    month "Apr" = return 4
-    month "May" = return 5
-    month "Jun" = return 6
-    month "Jul" = return 7
-    month "Aug" = return 8
-    month "Sep" = return 9
-    month "Oct" = return 10
-    month "Nov" = return 11
-    month "Dec" = return 12
+    month "Jan" = pure 1
```

Comment:
itching to simplify this function. 😅

Reply:
how can it be simpler than this?

Reply:
I'm not sure if simpler, but one idea would be using a sum type and extracting the `pure` to remove repetition. Just idle thoughts, not worth pursuing, I think.